VCFv4.4.tex

```diff
@@ -625,6 +625,41 @@ \section{INFO keys used for structural variants}
 \end{verbatim}
 \normalsize
 
+\footnotesize
+\begin{verbatim}
+##INFO=<ID=SVCLAIM,Number=A,Type=String,Description="Claim made by the structural variant call. Valid values are CN, BP, CNBP for copy number, breakpoint and both respectively.">
```

Comment:
CN, BP, CNBP should be changed to D, J, DJ, as well as their respective explanations